src/main/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheck.java

```diff
@@ -103,7 +103,7 @@ public UniquePropertiesCheck() {
     }
 
     @Override
-    @SuppressWarnings({"determinism:argument.type.incompatible","determinism:method.invocation.invalid"})  // true positive; logging: iteration over OrderNonDet collection
+    @SuppressWarnings({"determinism:argument.type.incompatible","determinism:method.invocation.invalid"})  // false positive; logging: iteration over OrderNonDet collection; messages logged in a SortedSet.
```

Comment:
When you changed this from "true positive" to "false positive", why did you retain "logging: iteration over OrderNonDet collection"?  That doesn't seem relevant to the false positive.  The same comment relates to other changes.